easybuild/easyconfigs/s/SQLite/SQLite-3.8.8.1-intel-2015a.eb

```diff
@@ -0,0 +1,40 @@
+##
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+#
+# Copyright:: Copyright 2012-2014 Uni.Lu/LCSB, NTUA
+# Authors::   Fotis Georgatos <fotis@cern.ch>
+# License::   MIT/GPL
+# $Id$
+#
+# This work implements a part of the HPCBIOS project and is a component of the policy:
+# http://hpcbios.readthedocs.org/en/latest/
+##
+
+easyblock = 'ConfigureMake'
+
+name = 'SQLite'
+version = '3.8.8.1'
+
+homepage = 'http://www.sqlite.org/'
+description = 'SQLite: SQL Database Engine in a C Library'
+
+toolchain = {'name': 'intel', 'version': '2015a'}
+
+# eg. http://www.sqlite.org/2014/sqlite-autoconf-3080600.tar.gz
+source_urls = ['http://www.sqlite.org/2015/']
+version_numbers = version.split('.')
+main_version = version_numbers.pop(0)
+version_str = '%s%s' % (main_version, ''.join('%02d' % int(x) for x in version_numbers))
```

Comment:
I don't consider this good style, applying splices directly to the result of methods convolutes code.  This only serves to confuse future generations.  Nevertheless...